Cargo.toml

```diff
@@ -51,13 +51,16 @@ bb8-postgres = "0.8"
 rusqlite = { version = "0.31.0", features = ["bundled"] }
 tokio-rusqlite = "0.5.1"
 mysql_async = { version = "0.34.1", features = ["native-tls-tls", "chrono"] }
-tokio-postgres = { version = "0.7.10", features = ["with-chrono-0_4", "with-uuid-1"] }
-clickhouse-rs = { git = "https://github.com/spiceai/clickhouse-rs.git", branch = "async-await", features = [
+tokio-postgres = { version = "0.7.10", features = [
+    "with-chrono-0_4",
+    "with-uuid-1",
+] }
+clickhouse-rs = { git = "https://github.com/spiceai/clickhouse-rs.git", tag = "0.2.0", features = [
```

Comment:
This needs to have the commits from https://github.com/spiceai/clickhouse-rs/tree/async-await-plus-hello-unwrap as well, or we will regress https://github.com/spiceai/spiceai/issues/1227